.github/workflows/Documentation.yml

```
@@ -2,17 +2,22 @@ name: Documentation
 
 on:
   push:
-    branches:
-      - 'master'
     tags: '*'
   pull_request:
 jobs:
   Documenter:
     name: Documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: julia-actions/julia-buildpkg@latest
+      - uses: actions/checkout@v3
+      - name: "Monorepo setup"
+        # also dev OndaEDFSchemas.jl for pull request builds
+        if: ${{ github.event_name == 'pull_request' }}
```

Documentation built in PRs and on `main` will use the version of OndaEDFSchemas.jl used in this commit while the documentation built for a specific version will use only a released version of OndaEDFSchemas.jl:
```suggestion
        # Only use a released version of OndaEDFSchemas.jl when creating documentation for a tag
        if: ${{ github.event_name != 'push' || !startsWith(github.ref, 'refs/tags') }}
```